docs/en/writing-running-appium/android/android-appbundle.md

```diff
@@ -0,0 +1,51 @@
+## How To Test Android App Bundle
```

Comment:
I think it makes sense to also add https://developer.android.com/studio/command-line/bundletool link into this article